lib/pronto/formatter/git_formatter.rb

```diff
@@ -79,6 +81,9 @@ def new_comments(messages, patches)
       end
 
       def remove_duplicate_comments(old_comments, new_comments)
+      puts "REMOVE DUPLICATE COMMENTS"
```

Comment:
- Layout/IndentationWidth: Use 2 (not 0) spaces for indentation.
- Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.